src/test/java/io/jenkins/pluginhealth/scoring/PluginServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package io.jenkins.pluginhealth.scoring;
+
+import io.jenkins.pluginhealth.scoring.model.Plugin;
+import io.jenkins.pluginhealth.scoring.repository.PluginRepository;
+import io.jenkins.pluginhealth.scoring.service.PluginService;
+
+import static org.junit.Assert.assertEquals;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.util.TestPropertyValues;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static org.mockito.Mockito.mock;
+
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@ContextConfiguration(initializers = PluginServiceTest.DockerPostgresDatasourceInitializer.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@Testcontainers
+@RunWith(MockitoJUnitRunner.class)
+public class PluginServiceTest {
```

Comment:
I chose to go with the Integration test rather than the unit test because I had trouble finding "what" and "how" to mock the pluginRepository and pluginService methods.

Then I tried to run this test class and I'm getting this error:
`org.junit.runners.model.InvalidTestClassError: Invalid test class 'io.jenkins.pluginhealth.scoring.PluginServiceTest':`
`  1. Test class should have exactly one public zero-argument constructor`